mix.exs

```diff
@@ -35,8 +35,8 @@ defmodule OMG.Umbrella.MixProject do
         git: "https://github.com/omisego/ex_unit_fixtures.git", branch: "feature/require_files_not_load", only: [:test]
       },
       {:ex_doc, "~> 0.20.2", only: :dev, runtime: false},
-      {:statix, "~> 1.1"},
-      {:appsignal, "~> 1.9"},
+      {:telemetry_poller, "~> 0.4.0"},
+      {:telemetry_metrics_statsd, "~> 0.2.0"},
```

Comment:
whats the reason for this change?

Reply:
You suggested using telemetry, and these are library, which is strongly related to telemetry (they have a common author).
They also have the same effect as statix.